### DISCUSSION TITLE
Migrating from `@tanstack/react-location` to `@tanstack/react-router`

### DISCUSSION THREAD

**Discussion by hckhanh** [TanStack/router#450] [Q&A]
I am using `@tanstack/react-location` for my enterprise product.
Is there any documentation that guides me to migrate to `@tanstack/react-router`?

**Comment by diego-bevilaqua86** [comment]
That's something I'd like to know too. I'm actually starting a new project, and even though everything points to using @tankstack/react-router, I still very much prefer the @tanstack/react-location API. Is location gonna be discontinued / deprecated?

**Reply by hckhanh** [reply]
Yup, I wonder if react-router will be the successor of location or not. Is there any deprecation for location?
